alpine/entry.sh

```diff
@@ -24,15 +24,35 @@ mount --move /tmp /dev
 ln -sf /dev/pts/ptmx /dev/ptmx
 
 mount -t debugfs nodev /sys/kernel/debug
-udevd & 
-udevadm trigger &> /dev/null
+
+if [ "$INITSYSTEM" = "on" ]; then
+	GREEN='\033[0;32m'
+	echo -e "${GREEN}OpenRC init system enabled."
+	env >> /etc/docker.env
+	printf '#!/bin/bash\n' > /etc/resinApp.sh
+	printf '%q ' "$@" >> /etc/resinApp.sh
+	chmod +x /etc/resinApp.sh
+
+	touch /var/log/resinapp.log
 	
-CMD=$(which $1)
-# echo error message, when executable file doesn't exist.
-if [  $? == '0' ]; then
-	shift
-	exec "$CMD" "$@"
+	sed -i -e s~#{DIR}~"$(pwd)"~g \
+		-e s~#{ENV}~"$(env | sed '/PATH=/d'  | tr '\n' ' ' | xargs printf '--env %s ')"~g /etc/init.d/resin
+
+	rc sysinit --quiet
+	rc default 
+	tail -f /var/log/resinapp.log
```

Comment:
systemd also needs to run as PID 1. The problem is that you're running `/sbin/init`, which starts a child process. What you want is `exec /sbin/init` which will replace the current process with openrc, and therefore it will be PID 1